WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/data/WatchFaceRepository.kt

```diff
@@ -34,42 +33,32 @@ class WatchFaceRepository(
 ) {
 
     // [AnalogWatchFaceEntity] properties/operations:
-    val allAnalogWatchFaces: Flow<List<AnalogWatchFaceEntity>> = analogWatchFaceDao.getAll()
+    suspend fun getAnalogWatchFace(analogWatchFaceId: Int) =
+        analogWatchFaceDao.get(analogWatchFaceId)
 
-    @WorkerThread
-    suspend fun insertAnalogWatchFace(analogWatchFace: AnalogWatchFaceEntity) {
-        analogWatchFaceDao.insert(analogWatchFace)
-    }
+    fun getAnalogWatchFaceAndStylesAndDimensions(analogWatchFaceId: Int) =
+        analogWatchFaceDao.getAnalogWatchFaceAndStylesAndDimensions(analogWatchFaceId)
 
     @WorkerThread
```

Comment:
You removed the annotation for one of the methods, but left it in for the others. I'd expect these to be non-necessary.

Reply:
I would remove it.

Reply:
Removed.